muxer.go

```diff
@@ -0,0 +1,411 @@
+package astits
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"github.com/asticode/go-astikit"
+	"io"
+)
+
+const (
```

Comment:
Is there any gain to export those constants?

Reply:
After some consideration I think there's none. Will change